hardhat.config.ts

```diff
@@ -106,6 +106,7 @@ module.exports = {
         settings: {
           evmVersion: "istanbul",
           metadata: { useLiteralContent: true },
+          optimizer: { enabled: true, runs: 200 }
```

Comment:
Is there any reason why we were not using optimiser ?

Reason why I enabled it is that manager.sol contract is too big and it would mean that I would have to split it

Reply:
I believe for security reasons. Optimizer is not as well trusted as the core compiler, from what I understand. We might have to split Manager (like we've e.g. done with Governance, splitting out Proposals).

Reply:
I split Manager.sol contract but it is not very pretty :-)